csrc/multidevice/collective.cpp

```diff
@@ -0,0 +1,199 @@
+// clang-format off
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+// clang-format on
+#ifdef USE_DISTRIBUTED
+
+#include <multidevice/collective.h>
+
+namespace nvfuser {
+namespace {
+static inline std::vector<std::vector<at::Tensor>> setBufList(
```

Comment:
Drop `static`.

Reply:
No problem. Could you explain why?

Reply:
The `static` specifier for a function is meant to make the scope of the function private to the compilation unit, which is achieved by the anonymous namespace, so it's just redundant. This is just more about our convention of preferring the anonymous namespace over static.